examples/3d-plot2.rs

```diff
@@ -0,0 +1,137 @@
+use gpui::{div, prelude::*, App, AppContext, View, ViewContext, WindowContext, WindowOptions};
+use parking_lot::RwLock;
+use plotters::coord::Shift;
+use plotters::drawing::DrawingArea;
+use plotters::prelude::*;
+use plotters_gpui::backend::GpuiBackend;
+use plotters_gpui::element::{PlottersChart, PlottersDrawAreaModel, PlottersDrawAreaViewer};
+use std::sync::Arc;
+
+fn pdf(x: f64, y: f64) -> f64 {
+    const SDX: f64 = 0.1;
+    const SDY: f64 = 0.1;
+    const A: f64 = 5.0;
+    let x = x / 10.0;
+    let y = y / 10.0;
+    A * (-x * x / 2.0 / SDX / SDX - y * y / 2.0 / SDY / SDY).exp()
+}
+
+struct MainViewer {
+    figure: View<PlottersDrawAreaViewer>,
+}
+
+impl MainViewer {
+    fn new(model: Arc<RwLock<PlottersDrawAreaModel>>, cx: &mut WindowContext) -> Self {
+        let figure = PlottersDrawAreaViewer::with_shared_model(model);
+
+        Self {
+            figure: cx.new_view(move |_| figure),
+        }
+    }
+}
+
+impl Render for MainViewer {
+    fn render(&mut self, cx: &mut ViewContext<Self>) -> impl IntoElement {
+        cx.defer(move |_, cx| {
+            cx.notify();
+        });
```

Comment:
This is need for animations. If no animation, can remove it

Reply:
This have animate.